src/main/kotlin/org/eclipse/tractusx/bpdmcertificatemanagement/entity/CertificateDB.kt

```diff
@@ -67,4 +68,10 @@ class CertificateDB(
     @Column(name = "uploader")
     var uploader: String?,
 
+    @Column(name = "document_id", unique = true)
+    val documentID: String = "DOC_ID_" + UUID.randomUUID().toString().substring(0, 16),
```

Comment:
Why have a fixed prefix there? The identifier is already in field document-id and with this it is clear it is the identifier of the document. 

Leaving out the prefix has the added advantage that the field can be of type UUID. Since postgres understands UUID you can also have a UUID column in the database. 

Also substring to 16 characters from 36 really reduces the size of possible random values making collisions more likely. Why not just go with the full 36 as intended with uuidv4?

Reply:
Yeah, the objective of the prefix was to better identify the field. I changed it to type UUID, and with the 36 chars, as it will give less problems in the future.

Reply:
There is still the option to prefix the number in the DTO response. But to be honest, I feel its not worth the effort. UUIDs are very common identifiers and when it comes to the end user, a user given certificate name is maybe a better option for them. Then such names also don't need to be technical keys but are just plain speaking names